File: src/commands/source.js
const { SlashCommandBuilder } = require("discord.js");

module.exports = {
  data: new SlashCommandBuilder()
    .setName("source")
    .setDescription("Displays information about the source code of this bot."),
  async execute(interaction) {
    await interaction.reply("TeX is an open-source project made specifically for the CSS Discord! You can see and contribute to the source code at https://github.com/CSSUoB/TeX-Bot-JS");
  },
};

File: src/cogs/commands/source.py
from discord import ApplicationContext, Bot
from discord.ext import commands


class Source(commands.Cog):
    def __init__(self, bot: Bot):
        self.bot: Bot = bot

    @commands.slash_command(description="Displays information about the source code of this bot.")
    async def source(self, ctx: ApplicationContext):
        await ctx.respond("TeX is an open-source project made specifically for the CSS Discord! You can see and contribute to the source code at https://github.com/CSSUoB/TeX-Bot-Py")


def setup(bot: Bot):
    bot.add_cog(Source(bot))
